addons/netfox/rollback/rollback-synchronizer.gd

```diff
@@ -83,10 +84,9 @@ func process_authority():
 	# Only record input that is our own
 	for property in input_properties:
 		var pe = _property_cache.get_entry(property)
+		_record_input_props.push_back(pe)
 		if pe.node.is_multiplayer_authority():
 			_auth_input_props.push_back(pe)
-		else:
-			_record_input_props.push_back(pe)
```

Comment:
Not critical enough to backport it to #251 
From what I understood, `_record_state_props` records the properties of a node, regardless of authority.
`_record_input_props` was unused (yet I needed it for that exact purpose, to learn the properties of a node I don't have authority on, as the receiver), so I replicated the same as `_record_state_props`

But this opens a new issue. Why do `_auth_input_props` and `_auth_state_props` even exist?!
They are currently used exclusively as booleans, to get if authority or not.
And it's not like properties are added realtime (if they are, they could as well be added onto `_record_state_props` and `_record_input_props` without problems.

My suggestion is to straight up remove the variables `_auth_input_props` and `_auth_state_props` unless I am missing something